library/rtstartup/rsbegin.rs

```diff
@@ -72,6 +72,10 @@ pub mod eh_frames {
         bool char
     }
 
+    // todo: why is this needed?
+    #[cfg(not(bootstrap))]
+    impl_copy! { f16 f128 }
+
```

Comment:
Is this still needed/good to have even though the types aren't used in this file? Not sure if the impl here applies to everything `no_core` even beyond this crate

Reply:
This is not needed here unless it is used in this specific file.

Reply:
Thanks, removed